website/docs/talent_development.md

```diff
@@ -28,275 +28,60 @@ There are two types of review cycles at dbt Labs.
-    2. What do you wish you had achieved but weren't able to? What were the blockers or challenges you faced?
-    3. What is the most motivating and least motivating part of your role?
-    4. What ratio of People Management : Individual Contributor work do you feel you do on average? Do you want to continue in this ratio?
-    5. What do you want to focus on developing over the next 6-12 months?
-    6. What can your manager do to continue to or better support you?
-    7. What can you do to continue to or better support your team?
-    8. How would you rate your performance as a manager this past half?
-        1. Needs Development/Meets Expectations/Exceeds Expectations/Sets a new standard
-- **Peer Feedback**
-    1. How has this person’s work impacted your ability to deliver on your goals this past 1-2 quarters?
-    2. What is one piece of constructive advice you would share with this person, and how would this change impact your work?
-    3. Which dbt Labs value(s) does this person embody? Which value(s) would you recommend they focus on in the coming half?
-- **Upwards Feedback**
-    1. What can your manager do to better support you in your work?
-    2. What areas of leadership development should your manager focus on that would positively impact the team's performance?
-    3. How does your manager advocate for your team?
-    4. Do you receive consistent and helpful feedback from your manager? Is it both constructive and affirmative, and what is the impact of this feedback for you?
-    5. How does your manager support your professional development and/or personal growth?
-    6. Which dbt Labs value(s) does your manager embody? Which value(s) would you recommend they focus on in the coming half?
-- **Manager Feedback to Employee**
-    1. How you rate this team member’s performance this past half?
-        1. Needs Development/Meets Expectations/Exceeds Expectations/Sets a new standard
-    2. What areas does this employee excel in? (*please reference some of the reasons for your performance rating)*
-    3. What was their biggest impact to the organization this past half? (*please reference some of the reasons for your performance rating)*
-    4. What was their biggest impact to their team’s success or accomplishments this past half?
-    5. What are leadership areas for this person to work on? *(please reference upwards feedback provided)*
-    6. How can you continue and better support this team member in their development?
-    7. Which dbt Labs value(s) does this person embody? Which value(s) would you like them to focus on in the coming half?
+#### Eligibility
+Employees must be in role for 4 or more of the 6 month evaluation period to be evaluated. 
 
 #### Performance Definitions
+![IMAGE](https://github.com/dbt-labs/handbook/blob/346a90fa6993732528a78c44aea15b988eb66a58/website/static/img/PR%20Update%202024.png)
```

Comment:
Hey @mirnawong1 that was going to be my first suggestion!

@r-scardoso can you adjust the link to the image so that it is a direct link to the file
e.g.
`![IMAGE](/website/static/img/PR%20Update%202024.png)`

If that doesn't work my next suggestion would be re-uploading the image with no spaces in the file name. Sometimes Vercel can be picky when it comes to spaces and/or uppercase letters in the file name.